src/components/C3.js

```diff
@@ -8,6 +8,7 @@ import * as translator from './Translator/C3'
 import { pageBlockBase, pageBlockConfigurators } from './PageBlocks/C3'
 
 import FileConfigurator from './Admin/EditorToolbar.c3'
+// import * as C3 from '@cortezaproject/corteza-vue'
```

Comment:
Guessing this is either supposed to be uncommented or not here

Reply:
I tried adding CItemPicker to compose's C3 arsenal but that's not possible for now because of the way we're importing C3 components. I've spoken to Denis and we're gonna leave this line commented for now.